### DISCUSSION TITLE
Starting talent points

### DISCUSSION THREAD

**Discussion by ghost** [FakeFishGames/Barotrauma#7203] [Ideas and suggestions]
Currently in missions mode there's no way to get levels or talents, but it could be interesting to let server hosts set an amount of starting talent points for players to let talents be used during missions, or maybe some way to earn XP mid-round instead of at the end of the round without already having talents.

**Comment by mLuby** [comment]
It'd be nice if the server lobby let the host specify starting XP. Until then, there are two workarounds:
1.  In the Sub Editor, add a bunch of commendations and medals to the sub's inventory so the captain can distribute them in the next game.
2. Mid-round the host can open the `f3` menu, `enablecheats` and then `spawnitem <commendation or medal> <player name>` to give that player the item. Repeat the last command (up arrow) until players hit the desired levels.
